nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2743,6 +2743,64 @@ nifi.content.repository.encryption.key.id=Key1
 nifi.content.repository.encryption.key=0123456789ABCDEFFEDCBA98765432100123456789ABCDEFFEDCBA9876543210
 ....
 
+
+=== HDFS Content Repository Properties
+
+This content repository uses the Hadoop FileSystem API to store FlowFile content. Because of this, it can be used to store content on the local disk and/or in one or more distinct HDFS clusters. It also has four different operating modes which are described below in the `nifi.content.repository.hdfs.operating.mode` property.
```

Comment:
Have any performance tests been done on this and if so for what kind of use cases/what performance?

Reply:
No, however I'm not really doing anything crazy with the Hadoop API. I unfortunately don't have access to a test cluster that would be able to exercise this well.